appveyor.yml

```diff
@@ -1,23 +1,44 @@
 image: Visual Studio 2017
 environment:
   OSENV: windows
-  OUTPUT: 'C:\projects\identity-wallet\sk-windows-' + $APPVEYOR_BUILD_VERSION + '.zip'
+  encryptedkey:
```

Comment:
this is the same type of setup as the ones on Travis?  needs the keys from inside Appveyor to work right?

Reply:
@altninja this is different. the values on the yml are encrypted and appveyor will decrypt it when running a push build. That is why the value is declared here. I am using the same travis keys  though. I deploy their values to GCS so I could also setup here and on cricle ci. cheers